schutzbot/Jenkinsfile

```diff
@@ -436,14 +436,14 @@ void run_tests(test_type) {
     if (test_type == 'base') {
         sh (
             label: "Base tests",
-            script: "schutzbot/run_base_tests.sh"
+            script: "/usr/libexec/tests/osbuild-composer/base_tests.sh"
```

Comment:
This breaks the idea of iterating over `/usr/libexec/tests/osbuild-composer/`, no? Shouldn't we move this somewhere else?

Reply:
Very good point! My idea was to move the test binaries that are not called directly by schutzbot somewhere else, so that schutzbot really just iterates over the scripts in `/usr/libexec/tests/osbuild-composer/*`. Now we have both the scripts in there and the binaries they call, which isn't quite right. Does that make sense? I'll add a patch to do that.

Reply:
Hm I don't understand. Where would that "somewhere else" be?

Reply:
`/usr/libexec/osbuild-copmoser-test/`. I'd move all the helper scripts there (some are currently in `/usr/libexec/osbuild-composer/`).

Reply:
Sounds good to me. Thanks for clarifying.